README.md

```diff
@@ -100,7 +100,7 @@ This project is licensed under the ASFv2 License. See the [LICENSE](https://gith
 
 ## Contributing  
   
-We welcome contributions from the community! If you'd like to add a new tool or improve an existing one, please check out our [CONTRIBUTING](https://github.com/The-AI-Alliance/gofannon/blob/main/CONTRIBUTING.md) guide for detailed instructions on how to get started.  
+We welcome contributions from the community! If you'd like to add a new tool or improve an existing one, please check out our [CONTRIBUTING]({{ "/CONTRIBUTING.md" | relative_url }})guide for detailed instructions on how to get started.  
```

Comment:
In the readme.md revert these to the full links. The Readme.md doesn't endup in the website, but it does power the pypi page where it needs the full links